.travis.yml

```diff
@@ -13,7 +13,7 @@ env:
   - secure: IbrybaUaJDiwNh8VodFfTIAdSw2Nl4DGRh2oDIO2vL8SPgGilwxY0go5DWrxAG0aP+YIXFkK+SpblFppQqqxnAoLVYmd7kFSJoOZraNqmcJqUYSzrHrWArmuiGzY8gMgOWTsnbzVVNRldN8tpoSwpQB/+YhCtbYb9eqvw+u5Izk=
   - secure: grvHX6c0ReQwRsuQZferI+uyGsKlubujURKuJB8coNdA9lgZcvf+nmCkNBpSML83hSjQkJUTt8XnucNNgL/1VxUhgy0O/wkTUEmcDAU1b80As8NJkbreJcc3HFASA68wou4HoJsFmJV2cpBYoAJJSX+HyT6mYd6pBh09tPfOeBg=
   matrix:
-  - TEST_COMMAND="run test:unit"
+  - TEST_COMMAND="run test:unit" MOCHA_TIMEOUT=10000
```

Comment:
Is this environment variable used by mocha, I thought it might just be our code that uses it.

Reply:
Its just our code.  I have it set the timeout in the gulpfile from parseInt(process.env['MOCHA_TIMEOUT'], 10) when present.

Reply:
Specifically in WCT

Reply:
Running polyserve test locally I see that is an expensive test:
```
      ✓ serves RequireJS library (365ms)
```
Also:
```
      ✓ caches html compilation results (471ms)
```
Upped the compile middleware test suite timeout to 10s.